api/SpeechRecognitionErrorEvent.json

```diff
@@ -38,16 +50,26 @@
           "safari_ios": {
             "version_added": false
           },
-          "samsunginternet_android": {
-            "prefix": "webkit",
-            "version_added": true,
-            "notes": "You'll need to serve your code through a web server for recognition to work."
-          },
-          "webview_android": {
-            "prefix": "webkit",
-            "version_added": true,
-            "notes": "You'll need to serve your code through a web server for recognition to work."
-          }
+          "samsunginternet_android": [
+            {
+              "version_added": false
+            },
+            {
+              "alternative_name": "webkitSpeechRecognitionError",
```

Comment:
I think we can remove the single `”version_added”: false` statement and just leave it to the one with the alternative name?

Reply:
> I think we can remove the single `”version_added”: false` statement and just leave it to the one with the alternative name?

Yep, OK — done

Reply:
Thank you!  We’ll also need to make it not an array.

Reply:
> Thank you! We’ll also need to make it not an array.

oops, yup — done